src/pages/index.tsx

```diff
@@ -1,15 +1,31 @@
 import Sponsors from "~/sections/Sponsors";
 import FAQ from "~/sections/FAQ";
 import Footer from "~/sections/Footer";
+import JoinGrowingCommunity from "~/sections/JoinGrowingCommunity";
 import Hero from "~/sections/Hero";
+import localFont from "next/font/local";
+
+const gochi = localFont({
```

Comment:
https://fontsource.org/fonts/gochi-hand/install Switch to this method of installation to be consistent with how other fonts are intalled